src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SyntaxTokenToSemanticTokensEditHelper.cs

```diff
@@ -1,99 +0,0 @@
-﻿// Copyright(c).NET Foundation.All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models;
-
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Services
-{
-    internal static class SyntaxTokenToSemanticTokensEditHelper
-    {
-        // The below algorithm was taken from OmniSharp/csharp-language-server-protocol at
-        // https://github.com/OmniSharp/csharp-language-server-protocol/blob/bdec4c73240be52fbb25a81f6ad7d409f77b5215/src/Protocol/Document/Proposals/SemanticTokensDocument.cs#L156
```

Comment:
Does https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/cgmanifest.json need to be reverted, or is it still used elsewhere?

Reply:
👍 Got it in the last commit.